### PR TITLE
SAK-46287: Tests & Quizzes: Question pool icons should be caret icons

### PR DESCRIPTION
**Pull request by stetsche**
Replace icons and fix spacing accordingly. Kept spacing images with smaller width to have a little bit wider focus indicator on the icon.

### CODE REVIEW

**Review by mpellicer — CHANGES_REQUESTED** [review]
Not sure about this? This is breaking previos changes and I'm not sure is not breaking the Javascript functions.

File: samigo/samigo-app/src/webapp/jsf/questionpool/poolTreeTable.jsp [hunk 1]
```diff
@@ -48,11 +48,11 @@
 <h:panelGroup styleClass="tier#{questionpool.tree.currentLevel}"  id="firstcolumn">
 <h:inputHidden id="rowid" value="#{questionpool.tree.currentObjectHTMLId}"/>
 <h:outputLink  title="#{questionPoolMessages.t_toggletree}" id="parenttogglelink"  onclick="toggleRows(this)" onkeypress="toggleRows(this)" value="#" styleClass="treefolder" rendered="#{questionpool.tree.hasChildList}" >
-    <h:graphicImage id="spacer_for_mozilla" style="border:0" height="14" width="30" value="/images/delivery/spacer.gif" />
-</h:outputLink>
-<h:outputLink title="#{questionPoolMessages.t_toggletreeEmpty}" id="togglelink"  value="#" styleClass="treedoc" rendered="#{questionpool.tree.hasNoChildList}" >
-    <h:graphicImage id="spacer_for_mozilla1" style="border:0" width="30" height="14"  value="/images/delivery/spacer.gif" />
+    <h:graphicImage id="spacer_for_mozilla" style="border:0" height="14" width="5" value="/images/delivery/spacer.gif" />
 </h:outputLink>
+<h:panelGroup styleClass="treedoc" rendered="#{questionpool.tree.hasNoChildList}" >
```
Comment (mpellicer) [inline samigo/samigo-app/src/webapp/jsf/questionpool/poolTreeTable.jsp:53]: Not sure about this.

Comment (stetsche) [reply samigo/samigo-app/src/webapp/jsf/questionpool/poolTreeTable.jsp:53]: What previous changes are broken here? I don't see Javaskript breaking here (unintentionally). This change is for pools without subpools. There shouldn't be be an icon. To also make it not focusable and clickable, i removed the link here.

Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/questionpool/poolTreeTable.jsp:53]: You are removing the title and the id, I think there's a Javascript event associated to the togglelink id.

title="#{questionPoolMessages.t_toggletreeEmpty}" id="togglelink"

Check the history of the file, I remember introducing the questionPoolMessages.t_toggletreeEmpty title for another Jira.

Comment (stetsche) [reply samigo/samigo-app/src/webapp/jsf/questionpool/poolTreeTable.jsp:53]: I don't find javascript related to that id. The property questionPoolMessages.t_toggletreeEmpty is still used in subpoolsTreeTable.jsp, so i would not remove it.

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thanks!

**Pull request merged**